packages/web-js-sdk/src/enhancers/withFingerprint/constants.ts

```diff
@@ -3,10 +3,12 @@ import { IS_BROWSER } from '../../constants';
 const FINGERPRINT_PUBLIC_KEY = 'fingerprint.public.key';
 const FINGERPRINT_ENDPOINT_URL = 'fingerprint.endpoint.url';
 
-/** Fingerprint.js custom API endpoint */
+/** Fingerprint.js cloudflare integration */
 export const FP_EP_URL =
   (IS_BROWSER && localStorage?.getItem(FINGERPRINT_ENDPOINT_URL)) ||
```

Comment:
just noting that this is somewhat duplication from core-js const (api.descope.com) 

I think its ok, just worth tracking in case change 👍